test/cypress/integration/widgets/secrets_spec.ts

```diff
@@ -1,7 +1,15 @@
 describe('Secret store management widget', () => {
     const secretName = 'secrets_test';
 
-    before(() => cy.activate().usePageMock('secrets').mockLogin().deleteSecrets(secretName));
+    before(() =>
+        cy
+            .activate()
+            .usePageMock('secrets')
+            .deleteSecretProviders()
+            .createSecretProvider({ name: 'Secret_Provider_1', type: 'vault', visibility: 'global' })
+            .mockLogin()
+            .deleteSecrets(secretName)
```

Comment:
I think more logically would be to prepare the API first and then login, so maybe:
```suggestion
            .deleteSecrets(secretName)
            .deleteSecretProviders()
            .createSecretProvider({ name: 'Secret_Provider_1', type: 'vault', visibility: 'global' })
            .usePageMock('secrets')
            .mockLogin()
```

Also, deleting secrets before deleting secret providers would make this test being more robust.

Reply:
done: https://github.com/cloudify-cosmo/cloudify-stage/pull/2370/commits/0ba89c8900177dcf45dd64741d1c744125bbc928